guidelines.md

```diff
@@ -76,7 +76,80 @@ The enterprise is based in UK, it accepts `GBP`, `EUR` and `USD` currencies \(an
 
 * **PlatformAddress** - `https://www.mews.li`
 * **ClientToken** - Will be provided to you by our integration team. For further information, please contact [integrations@mewssystems.com](mailto://integrations@mewssystems.com).
-* **AccessToken** - Depends on the enterprise, should be provided to you by the enterprise admin.
+* **AccessToken** - Will be provided to you via [email and your provided webhook](guidelines.md#AccessToken-delivery).
+
+### Integration event webhook
+
+The integrator may provide a webhook address to [integrations@mewssystems.com](mailto://integrations@mewssystems.com) where Mews will deliver data related to an integration [action](guidelines.md#Action).
+
+#### Response
+
+```javascript
+{
+  "Action": "IntegrationCreated",
+  "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+  "CreatedUtc": "2019-05-24T07:35:35Z",
+  "IsEnabled": true,
+  "Data": {
+    "Enterprise": {
+      "Id": "851df8c8-90f2-4c4a-8e01-a4fc46b25178",
+      "Name": "Connector API Hotel"
+    },
+    "Requestor": {
+      "Name": "Mews Integrations",
+      "Email": "integrations@mewssystems.com"
+    },
+    "Integration": {
+      "Id": "c66ef7b2-39d2-4632-943d-115ede9cb810",
+      "Name": "Example"
+    }
+  }
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Action` | string | required | The [Action](guidelines.md#Action) performed. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `CreatedUtc` | string | optional | Creation date and time of the integration in UTC timezone in ISO 8601 format. |
+| `IsEnabled` | string | optional | The state of the integration. |
+| `Data` | [Data](guidelines.md#data) | required | Data of the integration event. |
+
+#### Action
+
+* `IntegrationCreated` - A new integration has been created.
+* `IntegrationEnabled` - An integration has been enabled.
+* `IntegrationDisabled` - An integration has been disabled.
+* ...
+
+#### Data
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Enterprise` | [Enterprise](guidelines.md#Enterprise) | optional | Data of the integration event. |
+| `Requestor` | [Requestor](guidelines.md#Requestor) | optional | Data of the integration event. |
+| `Integration` | [Integration](guidelines.md#Integration) | required | Data of the integration event. |
+
+#### Enterprise
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Id` | string | required | Unique identifier of the enterprise. |
+| `Name` | string | required | Name of the enterprise. |
+
+#### Requestor
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Name` | string | required | Name of the user who requested the integration. |
+| `Email` | string | required | Email of the user who requested the integration. |
+
+#### Integration
```

Comment:
I don't think everything is provided in case of all actions. Data depend on action, so it should be reflected here, no?

Reply:
All is there, what do you see is missing?

Reply:
That the structure of the webhook is dynamic and depends on Action, e.g. in case of some actions, Requestor is not even present there. The documentation should ideally cover this.